include/action.h

```diff
@@ -18,11 +18,20 @@ struct action {
 	struct wl_list args;  /* struct action_arg.link */
 };
 
+struct action_query {
+	struct wl_list link;
+	const char *app_id;
```

Comment:
For window-rules we use “identifier” to cover xdg-shell’s app_id and xwayland’s WM_CLASS.

https://labwc.github.io/labwc-config.5.html#window_rules

https://github.com/labwc/labwc/blob/b149526d037b2de4809319cacb08c62dd3954b34/src/window-rules.c#L46

Reply:
Huh, totally forgot about XWayland TBH. Will update.

Reply:
Actually... do we want  to go forward with it? The rest of the code base uses app_id (e.g. window switcher) for both cases.

Reply:
The user-facing API (e.g. config file) is `identifier` and we should stick to that, what we do internally doesn't matter all that much.

Reply:
Are you sure? Because man clearly says app_id:

```
  <windowSwitcher show="yes" preview="yes" outlines="yes">
    <fields>
      <field content="type" width="25%" />
      <field content="app_id" width="25%" />
      <field content="title" width="50%" />
    </fields>
  </windowSwitcher>
```

Reply:
Hm.. good point. I was thinking about [window rules](https://labwc.github.io/labwc-config.5.html#window_rules).

Maybe we should change all window switcher examples to use "identifier" as well here (while still allowing "app_id" to not break older configs). Thoughts @johanmalm ?

Reply:
Yes, looks like my mistake using different currencies for window rules and window switcher. 

Agree with @Consolatis. Let’s use identifier, and patch window-switcher with backward compatible support.

Reply:
Yup, updated.